### ISSUE TITLE
 Unable to use pwndbg on Ubuntu16

### ISSUE THREAD

**Issue by mark0519** [pwndbg/pwndbg#1014]
### Description

I installed pwndbg using ``` ./setup.sh --with-python=usr/bin/python3```

After the installation is complete, when I use the gdb command, it will report an error

````shell
mark@ubuntu:~/pwndbg$ gdb
GNU gdb (Ubuntu 7.11.1-0ubuntu1~16.5) 7.11.1
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.  Type "show copying"
and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<http://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
<http://www.gnu.org/software/gdb/documentation/>.
For help, type "help".
Type "apropos word" to search for commands related to "word".
Traceback (most recent call last):
  File "/home/mark/pwndbg/gdbinit.py", line 24, in <module>
    import pwndbg # isort:skip
  File "/home/mark/pwndbg/pwndbg/__init__.py", line 21, in <module>
    import pwndbg.commands.context
  File "/home/mark/pwndbg/pwndbg/commands/context.py", line 141
    raise argparse.ArgumentError(banner_arg, f"banner can not be '{banner}'")
                                                                           ^
SyntaxError: invalid syntax
(gdb) 
````

````shell
(gdb) pi import sys
(gdb) pi print(sys.version)
3.5.2 (default, Jan 26 2021, 13:30:48) 
[GCC 5.4.0 20160609]
(gdb) 
````

### My setup

```Linux version 4.15.0-112-generic (buildd@lcy01-amd64-021) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.12)) #113~16.04.1-Ubuntu SMP Fri Jul 10 04:37:08 UTC 2020```

````bash
mark@ubuntu:~/pwndbg$ gdb -v
GNU gdb (Ubuntu 7.11.1-0ubuntu1~16.5) 7.11.1

mark@ubuntu:~/pwndbg$ python3 -V
Python 3.5.2

mark@ubuntu:~/pwndbg$ pip3 -V
pip 19.0.2 from /home/mark/.local/lib/python3.5/site-packages/pip (python 3.5)

CAN SOMEONE HELP ME？

````

**Comment by mark0519** [comment]
Thanks a lot for the help of my friend @ToBeatElite#4456 on pwndbg discord server, I have solved this problem.
![image](https://user-images.githubusercontent.com/73413552/162737806-574daf9c-32e0-4bbc-b323-0f042660c02b.png)
At first I added "banner = 'top' " before the check "if banner not in ('both', 'top', 'bottom', 'none'):", but strangely the problem persists. 
In the end I decided to comment out the check which didn't seem to be very important.
Now I can run pwndbg successfully
